client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
A better example of where this is more commonly used is `ActionTypes`:

https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=ActionTypes

>Do we really need to add another naming convention? [...] do we treat these variables in a significantly different enough way that it's justifiable to name them differently?

I agree with the worry of adding a new convention. I think there's a small argument to be made that it helps indicate that an object contains only constants. I'm leaning more towards keeping it simple and not advocating this convention, and in any case it's probably best kept for a discussion separate from these changes. I'll plan to update this to `handlers`.